content/en/continuous_delivery/features/changes_detection.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Changes Detection
+description: Learn how CD Visibility detects changes.
+further_reading:
+- link: "/continuous_delivery/deployments/"
+  tag: "Documentation"
+  text: "Learn about Deployment Visibility"
+- link: "/continuous_delivery/explorer"
+  tag: "Documentation"
+  text: "Learn how to query and visualize deployments"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility is in Preview. If you're interested in this feature, complete the form to request access.
+{{< /callout >}}
+
+## Overview
+
+Changes Detection allows Datadog to identify the commits introduced as part of a deployment. This is particularly valuable to:
+- Understand where specific changes have been deployed, such as tracking when updates reach the `production` environment.
+- Diagnose incidents related to a deployment by providing visibility into the exact changes introduced. This helps teams quickly pinpoint potential root causes and accelerate troubleshooting.
+
+To detect the changes deployed, Datadog runs the [`git log`][1] between the current deployment commit SHA and the previous deployment commit SHA. Merge commits are excluded from the computation.
+
+The deployed changes are visible inside any deployment execution of the [Deployment Executions page][2]. The **Code Changes** tab shows the previous deployment taken into consideration, and the changes detected between the two.
+
+{{< img src="continuous_delivery/features/code_changes_tab.png" alt="Code Changes tab for changes detection feature" style="width:100%;">}}
+
+Additionally, the **Deployments** column of the [Recent Code Changes][3] page displays the service and environment details for all deployments that included a specific commit. This view provides a quick way to understand if and where your changes are deployed.
+Hovering over the service value reveals whether the deployment has reached all expected environments, based on where the service is typically deployed.
+
+{{< img src="continuous_delivery/features/recent_code_changes_deployments.png" alt="Showing deployments in Recent Code Changes page" style="width:100%;">}}
+
+Changes are only detected for deployments that:
+- Have a service and an environment. Additionally, the service needs to have the file path specs defined in Software Catalog (see the [setup instructions](#specify-service-file-path-patterns) for more information).
+- Have a repository URL and a commit SHA defined.
```

Comment:
```suggestion
- Have a service and an environment.
- Have a repository URL and a commit SHA defined.
- The service has file path specs defined in Software Catalog (see the [setup instructions](#specify-service-file-path-patterns) for more information).
```

Reply:
Maybe you can switch 1 and 2 so that the 2 service requirements are close to the other